data/quick-builds/rebel-alliance.json

```diff
@@ -0,0 +1,53 @@
+{
+  "quick-builds": [
+    {
+      "threat": 2,
+      "pilots": [
+        {
+          "pilot": "bladesquadronveteran",
```

Comment:
I'd rename this field to `id` to keep it consistent with the xws spec.

Reply:
`id` or `xws`?

Reply:
`id` or `xws`?

Reply:
It would be id. Have a look at the "Pilot Data Format" in the xws-spechttps://github.com/elistevens/xws-spec